finagle-postgres-shapeless/src/test/scala/com/twitter/finagle/postgres/generic/QuerySpec.scala

```diff
@@ -26,8 +27,9 @@ class QuerySpec extends FreeSpec with Matchers with MockFactory {
 
   val client = new PostgresClient {
 
-    def prepareAndQuery[T](sql: String, params: Param[_]*)(f: (Row) => T): Future[Seq[T]] =
+    def prepareAndQueryToStream[T](sql: String, params: Param[_]*)(f: (Row) => T): Future[AsyncStream[T]] =
```

Comment:
Hmm, AsyncStream already wraps a Future. Its possible to construct an `AsyncStream[T]` directly from `Future[Seq[T]]` without wrapping it in the extra future